locksmithing.lic

```diff
@@ -44,7 +45,7 @@ class Locksmithing
         end
         DRC.bput("close my #{@box}", 'You close', 'That is already', "You can't close that")
       else
-        case DRC.bput("get my #{@box}", 'You get a', 'What were')
+        case DRC.bput("get my #{@box} from #{@picking_lockbox_container}", 'You get a', 'What were')
```

Comment:
Will this script break for anyone that gets the new changes but hasn't defined `picking_lockbox_container` yet?

Reply:
They will receive the error Daily Trainer not found! Where is it?. Right now, is just `case DRC.bput("get my #{@box}", 'You get a', 'What were')`.